### ISSUE TITLE
Continue Model Training?

### ISSUE THREAD

**Issue by holycowdude** [MRzzm/DINet#37]
Is it possible to add a feature or change some code please so we can continue training?

Currently if the training crashes or its stopped we cant continue and have to retrain from the start of that step

**Comment by Inferencer** [comment]
> 

I dont think you can but i'll look into it, what's the cuase of the crash? and are you running local or on a cloud service?

**Comment by holycowdude** [comment]
Running locally, sometimes when training on GPU with a batch size which is too high it crashes with an out of memory error after some time... so it would be perfect if it could continue training?

**Comment by Inferencer** [comment]
> Running locally, sometimes when training on GPU with a batch size which is too high it crashes with an out of memory error after some time... so it would be perfect if it could continue training?

oh I see, I dont belive you can but best to leave this open just in case,

 if you know how to setup on a cloud service I could help finance it depending on what dataset your trying to use, I asked another dev to do it but he's busy currently, I am getting fantastic results currently but would like to train on spesific in the wild vids add me on discord if you do `Inferencer#3759`

**Comment by holycowdude** [comment]
That sounds great, I can look at the cloud service.
Can I contact you by email please as I have a few questions?

**Comment by Inferencer** [comment]
> That sounds great, I can look at the cloud service. Can I contact you by email please as I have a few questions?

<details> 
  <summary> Email </summary>
    Jack.wharton.work@gmail.com
</details>

**Comment by ttkrpink** [comment]
will this config let you continue train with the last frame trained pth file?
```
--pretrained_frame_DINet_path
```
